.github/workflows/main.yml

```diff
@@ -12,7 +16,7 @@ jobs:
         repo: [ 'tarantool/tarantool']
     runs-on: ['self-hosted', 'multivac-crawler']
     env:
-      LOG_STORAGE_BUCKET_URL: ${{ secrets.LOG_STORAGE_BUCKET_URL }}
+      TOKEN: ${{ secrets.TOKEN }}
```

Comment:
It will be better to name it `MULTIVAC_GITHUB_TOKEN`.

Reply:
Done